.github/workflows/container-ingestion-images-cd.yml

```diff
@@ -0,0 +1,144 @@
+#
+# Build and push PostHog and PostHog Cloud container images
+#
+# - posthog_build: build and push the PostHog container image to DockerHub
+#
+# - posthog_cloud_build: build the PostHog Cloud container image using
+#   as base image the container image from the previous step. The image is
+#   then pushed to AWS ECR.
+#
+name: Ingestion Container Images CD
+
+on:
+    push:
+        branches:
+            - master
+    paths:
+      - 'plugin-server/**'
+
+concurrency: ${{ github.workflow }} # ensure only one of this runs at a time
+
+jobs:
+    posthog_build:
+        name: Build and push PostHog
+        if: github.repository == 'PostHog/posthog'
+        runs-on: ubuntu-latest
+        permissions:
+            id-token: write # allow issuing OIDC tokens for this workflow run
+            contents: read # allow at least reading the repo contents, add other permissions if necessary
+            packages: write # allow push to ghcr.io
+
+        steps:
+            - name: Checkout code
+              uses: actions/checkout@v3
+
+            - name: Update git SHA
+              run: echo "GIT_SHA = '${GITHUB_SHA}'" > posthog/gitsha.py
+
+            - name: Set up Docker Buildx
+              uses: docker/setup-buildx-action@v2
+
+            - name: Set up QEMU
+              uses: docker/setup-qemu-action@v2
+
+            - name: Set up Depot CLI
+              uses: depot/setup-action@v1
+
+            - name: Login to DockerHub
+              uses: docker/login-action@v2
+              with:
+                  username: ${{ secrets.DOCKERHUB_USERNAME }}
+                  password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+            - name: Build and push container images
+              id: build
+              uses: depot/build-push-action@v1
+              with:
+                  project: x19jffd9zf # posthog
+                  buildx-fallback: true # fallback to using 'docker buildx build' if 'depot build' is unable to acquire a builder connection
+                  cache-from: type=gha # always pull the layers from GHA
+                  cache-to: type=gha,mode=max # always push the layers to GHA
+                  push: true
+                  tags: posthog/posthog:latest
```

Comment:
this will be competing with the other workflow?

Reply:
How about we just build the plugin-server part of the Dockerfile and use that here?

Reply:
We could also do the same for the other build workflow but the other way around, i.e. not include the plugin server

Reply:
anyway, I think that becomes a little more complex for this PR

Reply:
@hazzadous The change you suggested is [here](https://github.com/PostHog/posthog/pull/16141/files#diff-78b403df46a7dc1a8a62a3d0cc6fb6acc4df6be3685f82d0b02fd17a28446c4dR16)

Is that ok?

Reply:
Yeah maybe.

I was more thinking docker images would be separate.

But the real point these two workflows can run at once, and they both have `posthog/posthog:latest`

Reply:
I updated the concurrency and that will ensure that they do not run in parallel.

Reply:
tbh I think until we have separate docker images, it seems like the previous version was better?

But I'll let @ellie pitch in here.